reactioncommerce/reaction
# Fix #3652 Publish shops of user

Closes #3652

### To test
1. Run `reaction`.
1. Login as admin and enable Marketplace.
2. Invite a marketplace admin.
1. Using Incognito to open the link in the invitation.
1. Observe no error in browser console and the shop name will be available in the navigation bar.
